exercises/object-oriented-programming/src/main/java/com/cbfacademy/accounts/Account.java

```diff
@@ -0,0 +1,28 @@
+package com.cbfacademy.accounts;
+
+public class Account {
+    protected double balance;
+    private int accountNumber;
+
+    public Account(double balance, int accountNumber) {
+        this.balance = balance;
+        this.accountNumber = accountNumber;
+    }
+
+    // 
+
+    public void deposit(double depositAmount) {
+        this.balance += depositAmount;
+    }
+
+    public void withdraw(double withdrawalAmount) {
```

Comment:
What might a bank account owner expect to get if they make a successful withdrawal? Say, at an ATM?